variables.tf

```diff
@@ -28,7 +28,7 @@ variable "system_user" {
 
 variable "use_ssh_agent" {
   type        = bool
-  default     = "true"
+  default     = "false"
```

Comment:
I would like to keep using ssh-agent by default... Would be a breaking change !

Reply:
ok, all done